doc/sphinx/README.template.rst

```diff
@@ -290,7 +297,24 @@ Emacs
 
 The ``dev/tools/coqdev.el`` folder contains a convenient Emacs function to quickly insert Sphinx roles and quotes.  It takes a single character (one of ``gntm:```), and inserts one of ``:g:``, ``:n:``, ``:t:``, or an arbitrary role, or double quotes.  You can also select a region of text, and wrap it in single or double backticks using that function.
 
-Use the following snippet to bind it to :kbd:`F12` in ``rst-mode``::
+Use the following snippet to bind it to `F12` in ``rst-mode``::
```

Comment:
@jfehrle Why having changed this? The display was better [before](https://github.com/coq/coq/tree/efe7108a0ae32faeb674cc5d97e6fa955a1dccd8/doc/sphinx#emacs) than [after](https://github.com/coq/coq/tree/3bd2722fb53552f45a25e6bc0a03a9ab0517485f/doc/sphinx#emacs).

Reply:
It was generating an error.  See https://github.com/coq/coq/pull/10424#issuecomment-505273368 above.  I took @cpitclaudel's suggestion on the fix.

Reply:
OK not important enough to figure out why this error was new.